src/server.tsx

```diff
@@ -408,7 +415,11 @@ const createEndpointForModule = (moduleInfo: ModuleInfo): void => {
     );
 };
 
-moduleInfos.forEach(createEndpointForModule);
+// Only serve the modules from this server when running locally (DEV).
+// In PROD/CODE we serve them from S3 via fastly.
+if (isDev) {
+    moduleInfos.forEach(createEndpointForModule);
+}
```

Comment:
This feels quite nice really. They're just static files in PROD so makes sense for express not to be serving them (even if all the caching already means there's no issue anyways)